contracts/account/proxy/src/contract.rs

```diff
@@ -57,6 +60,7 @@ pub fn instantiate(
 pub fn execute(deps: DepsMut, _env: Env, info: MessageInfo, msg: ExecuteMsg) -> ProxyResult {
     match msg {
         ExecuteMsg::ModuleAction { msgs } => execute_module_action(deps, info, msgs),
+        ExecuteMsg::ModuleActionWithData { msg } => execute_module_action_response(deps, info, msg),
```

Comment:
So we don't leave the possibility to execute multiple messages in case we want to retrieve the data ?

Reply:
No, because the response data is always overwritten by the last message. We could do a more complex multi-response capture but I really don't see the use-cases for that and it will introduce a bunch of complex encoding and decoding.